docs/requests.md

```diff
@@ -55,6 +55,16 @@ Headers are exposed as an immutable multi-dict.
 
 For example: `request.query_params['abc']`
 
+#### Cookies
+
+Cookies are exposed as an immutable dict.
+
+**Note**: Exception `KeyError` will be raised if nonexistant key is  accessed as `request.cookies['key']`.
+Instead cookies should be accessed with the `.get` method.
```

Comment:
I don't think we need to say this - it's just a regular dict interface, so that's expected behavior.